timemachine/cpp/src/context.hpp

```diff
@@ -34,6 +34,15 @@ class Context {
         int store_u_interval,
         int store_x_interval);
 
+    std::array<std::vector<double>, 2> local_md(
```

Comment:
For naming consistency with the global MD methods, could this be called `multiple_steps_local` or similar?

Reply:
Done in [574268f](https://github.com/proteneer/timemachine/pull/806/commits/574268fd50e5ae48be677b1522d637a33284bfba)